test/MVCFramework45.FunctionalTests/MVCFramework45.FunctionalTests.csproj

```diff
@@ -0,0 +1,71 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <VersionPrefix>2.0.0</VersionPrefix>
+    <TargetFrameworks>net451;netcoreapp1.0</TargetFrameworks>
```

Comment:
Sounds incorrect maybe. But wont be changing in this PR as  this is simply migrating project.json to .csproj.